app/controllers/users/two_step_verifications_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Users::TwoStepVerificationsController < ApplicationController
+  def new
+    redirect_to two_step_verification_path if current_user.two_factor_auth_enabled?
```

Comment:
Oh wait, that's not what this should redirect to. Then it's still not really clear to me. 😄